deployment/docker/sample.env

```diff
@@ -229,10 +229,25 @@ CUSTOM_SUBMISSION_URL=http://{your-ip-address}:{port}
 #the json data should be below format.
 
 #  `{
-#  "--navbar-background": "blue",
-#  "--navbar-items": "grey",
-#  "--navbar-active": "white"
-#  }`
+  "--navbar-active-submenu-bg-color": "#fbe9d0",
+  "--navbar-active-submenu-font-color": "#d79922",
+  "--navbar-menu-hover-bg-color": "#192d42",
+  "--navbar-main-menu-active-bg-color": "#446c7c",
+  "--navbar-main-menu-active-font-color": "#FFFFFF",
+  "--navbar-bg-color": "#83b2b7",
+  "--custom-logo-path": "https://logos-world.net/wp-content/uploads/2020/06/Amazon-Logo.png",
+  "--custom-title": "Amazon.in",
```

Comment:
--custom-title": **"Amazon.in"** ?

Reply:
it just a sample for reference @auslin-aot